### PR TITLE
Pagination should show correctly with fewer than 4 pages

### PR DESCRIPTION
**Pull request by hokei**
Fixes https://jira.nypl.org/browse/UX-234 and https://jira.nypl.org/browse/UX-230

## **This PR does the following:**
- The pagination component was showing strange behavior when there are fewer than 4 pages.  This has been fixed.  
Screen Shot 2021-01-28 at 3 32 30 PM
Screen Shot 2021-01-28 at 3 32 34 PM
Screen Shot 2021-01-28 at 3 32 38 PM

### Front End Review:
- [ ] View [the example in Storybook]()

### CODE REVIEW

File: src/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -113,25 +113,28 @@ export default class Pagination extends React.Component<PaginationProps> {
...
+            // second item will either be item 2 or an ellipse
+            pageStart > 3 ? "ellipse-start" : 2,
+            // Adding + 1 here since lodash range() doesn't include the last item passed to it
+            ...range(pageStart, pageEnd + 1),
+            // next to last item will either be an ellipse or the next to last page number
+            pageEnd < pageCount - 2 ? "ellipse-end" : pageCount - 1,
+            // list always ends at and displays the last page number in the pageCount
+            pageCount,
+          ];
+    console.log("truncatedList", truncatedList);
```
Comment (kaplanmichael) [inline src/components/Pagination/Pagination.tsx]: Please remove `console.log`.

**Review by kaplanmichael — CHANGES_REQUESTED** [review]
Looks good. The only issue I saw was that the focus outline sometimes gets cut off by the boundaries of the component:  
image

**Comment by hokei** [comment]
> Looks good. The only issue I saw was that the focus outline sometimes gets cut off by the boundaries of the component
This looks like a bigger issue unrelated to pagination.  (Here's something similar in `Link`) 
Screen Shot 2021-02-01 at 4 48 13 PM

Filed a DS ticket: https://jira.nypl.org/browse/DSD-30

**Review by kaplanmichael — APPROVED** [review]
Cool, thanks for adding that ticket.  LGTM.

**Pull request merged**